base/docs/helpdb/Base.jl

```diff
@@ -2083,11 +2083,18 @@ appended to an internal buffer of backtraces.
 :@profile
 
 """
-    extrema(itr)
+    extrema(itr) -> Tuple
```

Comment:
i'm sorry, but could you be more specific?  how do i "add the new signature to the rst"?  thanks.